NitroxServer/Communication/Packets/Processors/EquipmentAddItemPacketProcessor.cs

```diff
@@ -18,7 +18,16 @@ public EquipmentAddItemPacketProcessor(PlayerManager playerManager, PlayerData p
 
         public override void Process(EquipmentAddItem packet, Player player)
         {
-            playerData.AddEquipment(player.Name, packet.EquippedItemData);
+
+            if (packet.IsPlayerEquipment)
+            {
+                playerData.AddEquipment(player.Name, packet.EquippedItemData);
+            }
+            else
+            {
+                playerData.AddModule(packet.EquippedItemData);
```

Comment:
is this true that everything else is a module?  What about CyclopsDecoyLoadingTube?

Reply:
only player inventory type have own container everiting else have share container if CyclopsDecoyLoadingTube existe there is save on share container i tink that is the correct place
(im programing base a coop game)